guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -3,9 +3,9 @@
 
 Before installing {Project} in an IPv6 network, ensure that you meet the following requirements:
 
-* You must deploy an external DHCP IPv6 server as a separate, unmanaged service to bootstrap clients into GRUB2.
-GRUB2 configures IPv6 networking by using DHCPv6 or by assigning a static IPv6 address.
-This is required because the DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore the {SmartProxy} DHCP plugin that provides DHCP management is limited to IPv4 subnets.
+* You must deploy an external DHCPv6 server as a separate, unmanaged service to communicate with the network boot process and to manage IP address assignment.
+This is required because the {SmartProxy} DHCP plugin does not provide support for DHCPv6 management.
```

Comment:
```suggestion
* You must deploy an external DHCPv6 server and cofigure it manually to communicate with the network boot process and to manage IP address assignment because {Project} cannot integrate with a DHCPv6 server and manage its configuration.
```